models/review_test.go

```diff
@@ -105,3 +105,85 @@ func TestUpdateReview(t *testing.T) {
 	assert.NoError(t, UpdateReview(review))
 	AssertExistsAndLoadBean(t, &Review{ID: 1, Content: "Updated Review"})
 }
+
+func TestGetReviewersByPullID(t *testing.T) {
+	assert.NoError(t, PrepareTestDatabase())
+
+	issue := AssertExistsAndLoadBean(t, &Issue{ID: 2}).(*Issue)
+	user1 := AssertExistsAndLoadBean(t, &User{ID: 1}).(*User)
+	user2 := AssertExistsAndLoadBean(t, &User{ID: 2}).(*User)
+	user3 := AssertExistsAndLoadBean(t, &User{ID: 3}).(*User)
+	user4 := AssertExistsAndLoadBean(t, &User{ID: 4}).(*User)
+
+	// Create some reviews
+	expectedReviews := make(map[int64]*PullReviewersWithType)
+	// This one is from fixtures
+	expectedReviews[1] = &PullReviewersWithType{
+		User:              *user1,
+		Type:              ReviewTypeApprove,
+		ReviewUpdatedUnix: 946684810,
+	}
+
+	reviews := []CreateReviewOptions{
+		{
+			Content:  "New Review 1",
+			Type:     ReviewTypeComment,
+			Issue:    issue,
+			Reviewer: user1,
+		},
+		{
+			Content:  "New Review 3",
+			Type:     ReviewTypePending,
+			Issue:    issue,
+			Reviewer: user2,
+		},
+		{
+			Content:  "New Review 4",
+			Type:     ReviewTypeReject,
+			Issue:    issue,
+			Reviewer: user3,
+		},
+		{
+			Content:  "New Review 5",
+			Type:     ReviewTypeApprove,
+			Issue:    issue,
+			Reviewer: user4,
+		},
+	}
+	for _, test := range reviews {
+		rev, err := CreateReview(test)
+		assert.NoError(t, err)
+		// Only look for non-pending reviews, nor comments
+		if test.Type == ReviewTypeApprove || test.Type == ReviewTypeReject {
+			expectedReviews[test.Reviewer.ID] = &PullReviewersWithType{
+				User:              *test.Reviewer,
+				Type:              test.Type,
+				ReviewUpdatedUnix: rev.UpdatedUnix,
+			}
+		}
+	}
+
+	allReviews, err := GetReviewersByPullID(issue.ID)
+	assert.NoError(t, err)
+	assert.Equal(t, expectedReviews, allReviews)
+
+	// Add another one, this time overwriting a previously "pending" one
+	newReview := CreateReviewOptions{
```

Comment:
Just add two reviews of the same users to the fixtures, since that's what you're testing here. If not, please let me know.